constants/index.ts

```diff
@@ -262,4 +262,6 @@ export const MAX_DAILY_EMAILS = 100 // If changed, update the DB default accordi
 export const XEC_TX_EXPLORER_URL = 'https://explorer.e.cash/tx/'
 export const BCH_TX_EXPLORER_URL = 'https://blockchair.com/bitcoin-cash/transaction/'
 
-export const MAX_MESSAGES_TO_PROCESS_AT_A_TIME = 2
+export const MAX_MEMPOOL_TXS_TO_PROCESS_AT_A_TIME = 3
```

Comment:
Why not more?

Reply:
More can lead to the prisma error of  "too many db connections". This will only be used if two mempool txs arrive ALMOST at the same time, since they're processed really quickly. So this actually doesn't make much of a difference in payment processing time, only prevents the mentioned error.